app/webpacker/scripts/form-check-leave.js

```diff
@@ -0,0 +1,31 @@
+function FormCheckLeave($module) {
+  this.$module = $module;
+}
+
+FormCheckLeave.prototype.init = function() {
+  const $form = this.$module;
+  let $change = false;
+
+  if ($form) {
+    $form.addEventListener("submit", function() {
+      window.onbeforeunload = null;
+    });
+
+    $form.addEventListener("change", function() {
+      $change = true;
```

Comment:
Nice :+1: This works for me when testing locally.

I've found one small bug – when a user copies content from another course we add an onbeforeunload too:
https://github.com/DFE-Digital/manage-courses-frontend/blob/4d58255a5b3ba3c87d9f96b4ce16773480ab7751/app/webpacker/packs/application.js#L23-L25

This gets replaced with an empty function if nothing is changed now.

Steps to reproduce:
- Got to About this course
- Copy from another course (an orange box will show)
- When leaving this page (without making further changes), we need to warn of unsaved changes – this is because what was copied is not yet saved